packages/quantic/force-app/custom-token/main/default/classes/CustomTokenProvider.cls

```diff
@@ -0,0 +1,119 @@
+/**
+ * Example class showing off how to create your own TokenProvider,
```

Comment:
Not sure about the use of `off` here. I would suggest replacing `showing off` by `showing` or `explaining`.